runtime/commonMain/src/kotlinx/serialization/json/internal/JsonReader.kt

```diff
@@ -7,7 +7,7 @@ package kotlinx.serialization.json.internal
 import kotlinx.serialization.json.*
 import kotlinx.serialization.json.internal.EscapeCharMappings.ESCAPE_2_CHAR
 import kotlin.jvm.*
-import kotlin.native.concurrent.SharedImmutable
+import kotlin.native.concurrent.*
```

Comment:
Side note: maybe we should rename `JsonReader` to `JsonLexer` because `JsonReader` can be confused with existent `JsonParser`

Reply:
well, it's internal, so shouldn't be a big problem to keep it that way or to rename.
I'm fine with both